reporters/detail.js

```diff
@@ -92,7 +92,7 @@ const report = function (data, options) {
         if (action.action === 'update' || action.action === 'install') {
           const recommendation = getRecommendation(action, config)
           const label = action.resolves.length === 1 ? 'vulnerability' : 'vulnerabilities'
-          log(`\n\n# Run \`${Utils.color(recommendation.cmd, 'red', config.withColor)}\` to resolve ${action.resolves.length} ${label}`)
+          log(`\n\n# Run ${Utils.color(' ' + recommendation.cmd + ' ', 'inverse', config.withColor)} to resolve ${action.resolves.length} ${label}`)
```

Comment:
When this was red, it was `Utils.color(…, ['red', 'inverse'], configWithColor)`. (The console-control-strings color function can combine multiple color codes efficiently.)

Worth noting here is the added spacing around the command.